kalliope/trigger/snowboy/snowboy.py

```diff
@@ -29,26 +29,43 @@ def __init__(self, **kwargs):
 
         # get the sensitivity if set by the user
         self.sensitivity = kwargs.get('sensitivity', 0.5)
+        self.apply_frontend = kwargs.get('apply_frontend', False)
 
         # callback function to call when hotword caught
         self.callback = kwargs.get('callback', None)
         if self.callback is None:
             raise MissingParameterException("callback function is required with snowboy")
 
-        # get the pmdl file to load
-        self.pmdl = kwargs.get('pmdl_file', None)
-        if self.pmdl is None:
-            raise MissingParameterException("Pmdl file is required with snowboy")
-
-        self.pmdl_path = Utils.get_real_file_path(self.pmdl)
-        if not os.path.isfile(self.pmdl_path):
-            raise SnowboyModelNotFounfd("The snowboy model file %s does not exist" % self.pmdl_path)
-
-        self.detector = snowboydecoder.HotwordDetector(self.pmdl_path,
-                                                       sensitivity=self.sensitivity,
+        # get the keywords to load
+        self.keywords = kwargs.get('keywords', None)
+
+        self.pmdl_file = kwargs.get('pmdl_file', None)  # We notify the user that the pmdl_file parameter has been changed
+        if self.pmdl_file:
+            raise MissingParameterException('"pmdl_file" parameter is deprecated, please update your snowboy settings. \n Visit https://kalliope-project.github.io/kalliope/settings/triggers/snowboy/ for more information.')
+
+        if self.keywords is None:
+            raise MissingParameterException("At least one keyword is required with snowboy")
+
+        if self.keywords:
```

Comment:
no need to test it again here. It has been tested before

Reply:
That's true